### PR TITLE
New Java test

### PR DESCRIPTION
**Pull request by freyabyrne**
This is my first ever pull request, I'm so excited :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ Contributions are very welcome. Please see [CONTRIBUTING.md](https://github.com/
 - [Xfive](https://github.com/xfiveco/front-end-test-project) Front-end test project used in recruitment.
 - [Yodlr](https://github.com/yodlr/frontend-code-challenge) Code/design challenge for front end engineering applicants.
 
+## Java
+
+- [WorldTicket](https://bitbucket.org/worldticket/recruitment-tests/src/default/) Front-end test project used in recruitment.
```
Comment (zsoltime) [inline README.md]: It looks good but could you update the description, please? Something like "Create a simple web application for currency conversion." would be better.

**Review by zsoltime — CHANGES_REQUESTED** [review]
Great, you've just created a new heading. 👍 Could you add it to the contents list? 

The link text can be "Java" and the URL "#java". You can insert it between "HTML & CSS" and "JavaScript".

Thank you.

File: README.md (hunk 1 above)
Comment (freyabyrne) [reply README.md]: I've just updated it :)

**Review by zsoltime — APPROVED** [review]

**Comment by zsoltime** [comment]
Thank you. Nice job, just merged them 😉

**Comment by freyabyrne** [comment]
Thank you :)

**Pull request merged**